### PR TITLE
Fixed netfx System.Text.Encoding.Tests fails on non-English Windows

### PR DESCRIPTION
**Pull request by AlexRadch**
See https://github.com/dotnet/corefx/issues/28136

### CODE REVIEW

File: src/System.Text.Encoding/tests/Encoding/EncodingGetEncodingTests.cs [hunk 1]
```diff
@@ -99,10 +100,10 @@ public void TestEncodingNameAndCopdepageNumber()
         [Fact]
         public void GetEncoding_EncodingName()
         {
-            CultureInfo originalUICulture = CultureInfo.CurrentUICulture;
-            try
+            // Workaround issue: UWP culture is process wide
+            RemoteInvoke(() =>
             {
-                CultureInfo.CurrentCulture = new CultureInfo("en-US");
```
Comment (stephentoub) [inline src/System.Text.Encoding/tests/Encoding/EncodingGetEncodingTests.cs]: Why was this failing on a non-English system when the culture was being set to en-US?

Comment (AlexRadch) [reply src/System.Text.Encoding/tests/Encoding/EncodingGetEncodingTests.cs]: It was CurrentCulture but should be Current**UI**Culture. So I changed to Current**UI**Culture and added RemoteInvoke as Workaround UWP culture is process wide issue.

**Comment by AlexRadch** [comment]
@dotnet-bot test OSX x64 Debug Build

File: src/System.Text.Encoding/tests/Encoding/EncodingGetEncodingTests.cs (hunk 1 above)
Comment (stephentoub) [reply src/System.Text.Encoding/tests/Encoding/EncodingGetEncodingTests.cs]: Ok, thanks.

**Pull request merged**